### PR TITLE
av

### PR DESCRIPTION
**Pull request by angelicavalenta**
Client Side 

- I have been committing on the developer branch and am not sure this is correct
- Please review and advise if there are any best practices I can incorporate to the project 

Thank you

### CODE REVIEW

**Comment by arol** [comment]
Great description, but write a more meaningful title (you can edit it).

**Comment by arol** [comment]
When working on a feature you should create a branch called `feat/name` and commit/push on it. Then the pull request should be done from that feature branch into develop (this one is develop to master).

File: Seaworthy/src/app/account/account.component.ts [hunk 1]
```diff
@@ -1,15 +1,97 @@
...
 
-}
+  emailFormControl = new FormControl('', [
+    Validators.required,
+    Validators.pattern(EMAIL_REGEX)]);
+
+    stateCtrl: FormControl;
+    filteredStates: any;
+
+    states = [
```
Comment (arol) [inline Seaworthy/src/app/account/account.component.ts]: You can put this array data in another file and import it, is a better practice.

**Review by arol — APPROVED** [review]

**Comment by arol** [comment]
That new branch that you create should be created from develop.

Keep in mind all these comments for the next feature you implement!

**Comment by angelicavalenta** [comment]
THANK YOU for the feedback - I'll make adjustments. 

Get you some beach time - enjoy!

**Comment by arol** [comment]
hey, you can merge it already!

**Pull request merged**